packages/hardhat-core/test/internal/hardhat-network/stack-traces/test.ts

```diff
@@ -809,6 +809,11 @@ const solidityCompilers: SolidityCompiler[] = [
     compilerPath: "soljson-v0.8.17+commit.8df45f5f.js",
     latestSolcVersion: true,
   },
+  {
+    solidityVersion: "0.8.18",
+    compilerPath: "soljson-v0.8.18+commit.87f61d96.js",
+    latestSolcVersion: true,
```

Comment:
This is the only thing that caught my attention.

Reply:
Not sure what you mean. The only thing that comes to mind about this is that maybe we should remove the `latestSolcVersion` flag from a previous version (e.g., 0.8.14).

Right now we don't have a policy about this. Maybe something like "the latest 3 versions of the current minor, and the last version of the previous minor" is enough.

Reply:
To clarify: this flag is used to filter the tested versions by default, but we also have a workflow that runs the stack traces tests with all the configured versions. So this is a sort of `fast` vs `full` tests config.

Reply:
I removed the `latestSolcVersion` flag from 0.8.14 and 0.8.15.